### PR TITLE
etcdctl: improve error message on migration without v2 keys

### LINKED ISSUE #5478: migrate command on fresh cluster exit 1 with cryptic error
### What's the issue?

If I have a empty v2 store and do data migration, it will fail with error message:
```
Error:  Key not found (/1)
```

### How to reproduce?
Have etcd and etcdctl in $PATH.
```sh
etcd
cp -r default.etcd data
etcdctl migrate --data-dir=./data
```

### PR DESCRIPTION
**Pull request by heyitsanthony**
Fixes #5478

### CODE REVIEW

File: etcdctl/ctlv3/command/migrate_command.go [hunk 1]
```diff
@@ -194,6 +195,9 @@ func toTTLOptions(r *pb.Request) store.TTLOptionSet {
 func writeStore(w io.Writer, st store.Store) uint64 {
 	all, err := st.Get("/1", true, true)
 	if err != nil {
+		if eerr, ok := err.(*etcdErr.Error); ok && eerr.ErrorCode == etcdErr.EcodeKeyNotFound {
```
Comment (xiang90) [inline etcdctl/ctlv3/command/migrate_command.go]: should this be an error? or just exit since there is no keys to migrate?

**Comment by gyuho** [comment]
lgtm. defer to @xiang90 , @hongchaodeng

File: etcdctl/ctlv3/command/migrate_command.go [hunk 2]
```diff
@@ -194,6 +195,9 @@ func toTTLOptions(r *pb.Request) store.TTLOptionSet {
...
+			ExitWithError(ExitInvalidInput, fmt.Errorf("no v2 keys found"))
```
Comment (hongchaodeng) [inline etcdctl/ctlv3/command/migrate_command.go]: We shouldn't return error -- actually not exit 1, if there is no v2 keys.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**